go/pkg/db/queries/role_find_by_names.sql

```diff
@@ -0,0 +1,2 @@
+-- name: FindRolesByNames :many
+SELECT id, name FROM roles WHERE workspace_id = sqlc.arg('workspace_id') AND name IN (sqlc.slice('names'))
```

Comment:
why is `sql.slice('names')` wrapped in parentheses? Is that some sqlc BS restriction?